packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -75,6 +87,14 @@ export class DbSyncStakePoolProvider extends DbSyncProvider(RunnableModule) impl
     this.#epochMonitor = epochMonitor;
     this.#metadataService = metadataService;
     this.#paginationPageSizeLimit = paginationPageSizeLimit;
+    this.response = merge({
```

Comment:
Nice, the name confused me at first though
```suggestion
    this.responseConfig = merge({
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/609/commits/762a8ad009524278c13511468f978ff427f61983